packages/formstr-app/src/containers/CreateFormNew/components/FormDetails/index.tsx

```diff
@@ -64,9 +83,62 @@ export const FormDetails: React.FC<FormDetailsProps> = ({
     setSavedLocally(true);
   };
 
+  const saveToMyForms = async (
+    formAuthorPub: string,
+    formAuthorSecret: string,
+    formId: string,
+    relay: string,
+    viewKey?: string
+  ) => {
+    if (!userPub) return;
```

Comment:
should we show some message or is this an edge case?

Reply:
myforms section should only be triggered when the user is logged in, i'm not sure when this happens in the normal case so left it as is